packages/fluentui/react-northstar/test/specs/components/Accordion/Accordion-test.tsx

```diff
@@ -107,7 +132,7 @@ describe('Accordion', () => {
       const wrapper = mountWithProvider(<Accordion panels={panels} />);
       const accordion = wrapper.find(Accordion);
       getTitleButtonAtIndex(wrapper, 1).simulate('click');
-      expect(accordion.state('focusedIndex')).toEqual(1);
+      expect(getAccordionTitleAtIndex(accordion, 1)).toEqual(document.activeElement);
```

Comment:
check this out https://github.com/testing-library/jest-dom#tohavefocus :D

Reply:
Nice, changed all